### ISSUE TITLE
test_ndarray_methods crashes omniscidb server when CUDA enabled

### ISSUE THREAD

**Issue by pearu** [xnd-project/rbc#93]
```
max, max_empty, max_initial, min, min_empty, min_initial, mean:
NvidiaKernel.cpp:96 Check failed: cuLinkAddData_v2(link_state, CU_JIT_INPUT_PTX, static_cast<void*>(const_cast<char*>(ptx.c_str())), ptx.length() + 1, 0, 0, nullptr, nullptr) == CUDA_SUCCESS (218 == 0)

mean_empty_float, sum, sum_initial, prod, prod_initial:
NvidiaKernel.cpp:101 Check failed: cuLinkComplete(link_state, &cubin, &cubinSize) == CUDA_SUCCESS (999 == 0)

mean_empty_int:
NativeCodegen.cpp:868 invalid redefinition of function 'llvm.lifetime.start.p0i8'
```

`OmniSci Version: 5.4.0dev-20200604-fc4a06f9fc`

**Comment by pearu** [comment]
With https://github.com/omnisci/omniscidb-internal/pull/4714 we have for `max` etc
```
2020-08-10T16:37:31.320447 F 27570 4 NvidiaKernel.cpp:126 Check failed: cuLinkAddData_v2(link_state, CU_JIT_INPUT_PTX, static_cast<void*>(const_cast<char*>(ptx.c_str())), ptx.length() + 1, 0, 0, nullptr, nullptr) == CUDA_SUCCESS (218 == 0) ptxas application ptx input, line 453; error   : Call has wrong number of parameters
ptxas fatal   : Ptx assembly aborted due to errors
```

sum:
```
2020-08-10T16:39:35.083121 F 27739 6 NvidiaKernel.cpp:132 Check failed: cuLinkComplete(link_state, &cubin, &cubinSize) == CUDA_SUCCESS (999 == 0) error   : Undefined reference to 'calloc' in 'Input 2'
Aborted (core dumped)
```

**Comment by pearu** [comment]
Resolved via #209
